File: 프로그래머스/unrated/181921. 배열 만들기 2/배열 만들기 2.java
import java.util.ArrayList;
import java.util.Arrays;
import java.util.List;

class Solution {
    public static int[] solution(int l, int r) {
        List<Integer> answer = new ArrayList<>();
        
        for (int i=l; i<=r; i++) {
            String str = String.valueOf(i);
            for (int j=0; j<str.length(); j++) {
                if (str.charAt(j) != '0' && str.charAt(j) != '5') break;
                if (j == str.length()-1) {
                    answer.add(Integer.valueOf(str));
                }
            }
        }
         
        return answer.isEmpty() ? new int[]{-1} : answer.stream().mapToInt(Integer::valueOf).toArray();
    }
}

File: 프로그래머스/unrated/181921. 배열 만들기 2/배열 만들기 2.py
def solution(l, r):
    answer = []

    for n in range(l, r+1):
        i = 0
        for ch in str(n):
            if (ch != '0' and ch != '5'): break
            i += 1
            if i == len(str(n)):
                answer.append(n)

    return [-1] if not answer else answer
